src/panels/config/dashboard/ha-config-updates.ts

```diff
@@ -112,6 +116,17 @@ class HaConfigUpdates extends LitElement {
       ha-logo-svg {
         color: var(--secondary-text-color);
       }
+      .show-all {
+        cursor: pointer;
+        color: var(--primary-color);
+        margin: 4px 16px;
```

Comment:
Let's give this some more top and bottom margin

Reply:
Or better, use a button, so we also get feedback when you click it etc, and it is also a11y